frontend/src/components/AssignmentModal/VehicleConditionInfoCard.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <v-card-text>
+    <v-container>
+      <v-row>
+        <v-col cols="12">
+          <p class="text-subtitle-1">Incident information</p>

```

Comment:
Maybe use the h1 tag here. It will be semantically better

Reply:
Change to h2. h1 for 'assignment creation' title